### ISSUE TITLE
Modify user attribute (fullname)

### ISSUE THREAD

**Issue by ilgrosso** [ilgrosso/oldSyncopeIdM#234]
_From [mdisabat...@gmail.com](https://code.google.com/u/101550733072735377085/) on December 05, 2011 16:50:14_

What steps will reproduce the problem? 1. Modify fullname attribute of an user from console with embedded data.

What is the expected output?
The user has changed the fullname.

What do you see instead?
Error: {[DuplicateUniqueValue []]}

ERROR org.syncope.console.pages.panels.UserSummaryPanel - While creating or updating user
org.syncope.client.validation.SyncopeClientCompositeErrorException: {[DuplicateUniqueValue []], }
	at org.syncope.client.validation.SyncopeClientErrorHandler.handleError(SyncopeClientErrorHandler.java:58) ~[syncope-client-0.7-SNAPSHOT.jar:na]
	at org.springframework.web.client.RestTemplate.handleResponseError(RestTemplate.java:486) ~[spring-web-3.0.6.RELEASE.jar:3.0.6.RELEASE]
	at org.springframework.web.client.RestTemplate.doExecute(RestTemplate.java:443) ~[spring-web-3.0.6.RELEASE.jar:3.0.6.RELEASE]
	at org.springframework.web.client.RestTemplate.execute(RestTemplate.java:401) ~[spring-web-3.0.6.RELEASE.jar:3.0.6.RELEASE]
	at org.springframework.web.client.RestTemplate.postForObject(RestTemplate.java:279) ~[spring-web-3.0.6.RELEASE.jar:3.0.6.RELEASE]
	at org.springframework.web.client.RestTemplate$$FastClassByCGLIB$$aa4e9ed0.invoke(<generated>) ~[cglib-2.2.jar:3.0.6.RELEASE]
	at net.sf.cglib.proxy.MethodProxy.invoke(MethodProxy.java:191) ~[cglib-2.2.jar:na]
	at org.springframework.aop.framework.Cglib2AopProxy$CglibMethodInvocation.invokeJoinpoint(Cglib2AopProxy.java:688) ~[spring-aop-3.0.6.RELEASE.jar:3.0.6.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:150) ~[spring-aop-3.0.6.RELEASE.jar:3.0.6.RELEASE]
	at org.springframework.aop.support.DelegatingIntroductionInterceptor.doProceed(DelegatingIntroductionInterceptor.java:131) ~[spring-aop-3.0.6.RELEASE.jar:3.0.6.RELEASE]
	at org.springframework.aop.support.DelegatingIntroductionInterceptor.invoke(DelegatingIntroductionInterceptor.java:119) ~[spring-aop-3.0.6.RELEASE.jar:3.0.6.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172) ~[spring-aop-3.0.6.RELEASE.jar:3.0.6.RELEASE]
	at org.springframework.aop.framework.Cglib2AopProxy$DynamicAdvisedInterceptor.intercept(Cglib2AopProxy.java:621) ~[spring-aop-3.0.6.RELEASE.jar:3.0.6.RELEASE]
	at org.springframework.web.client.RestTemplate$$EnhancerByCGLIB$$feffd289.postForObject(<generated>) ~[cglib-2.2.jar:3.0.6.RELEASE]
	at org.syncope.console.rest.UserRestClient.update(UserRestClient.java:68) ~[UserRestClient.class:na]
	at WICKET_org.syncope.console.rest.UserRestClient$$FastClassByCGLIB$$98ecf4fe.invoke(<generated>) ~[cglib-2.2.jar:na]
	at net.sf.cglib.proxy.MethodProxy.invoke(MethodProxy.java:191) ~[cglib-2.2.jar:na]
	at org.apache.wicket.proxy.LazyInitProxyFactory$CGLibInterceptor.intercept(LazyInitProxyFactory.java:318) ~[wicket-ioc-1.5.3.jar:1.5.3]
	at WICKET_org.syncope.console.rest.UserRestClient$$EnhancerByCGLIB$$bac72eec.update(<generated>) ~[cglib-2.2.jar:na]
	at org.syncope.console.pages.UserModalPage$1.onSubmit(UserModalPage.java:287) ~[UserModalPage$1.class:na]
	at org.apache.wicket.ajax.markup.html.form.AjaxButton$1.onSubmit(AjaxButton.java:103) [wicket-core-1.5.3.jar:1.5.3]
	at org.apache.wicket.ajax.form.AjaxFormSubmitBehavior$1.onSubmit(AjaxFormSubmitBehavior.java:172) [wicket-core-1.5.3.jar:1.5.3]
	at org.apache.wicket.markup.html.form.Form.delegateSubmit(Form.java:1174) [wicket-core-1.5.3.jar:1.5.3]
	at org.apache.wicket.markup.html.form.Form.process(Form.java:838) [wicket-core-1.5.3.jar:1.5.3]
	at org.apache.wicket.markup.html.form.Form.onFormSubmitted(Form.java:762) [wicket-core-1.5.3.jar:1.5.3]
	at org.apache.wicket.ajax.form.AjaxFormSubmitBehavior.onEvent(AjaxFormSubmitBehavior.java:158) [wicket-core-1.5.3.jar:1.5.3]
	at org.apache.wicket.ajax.AjaxEventBehavior.respond(AjaxEventBehavior.java:166) [wicket-core-1.5.3.jar:1.5.3]
	at org.apache.wicket.ajax.AbstractDefaultAjaxBehavior.onRequest(AbstractDefaultAjaxBehavior.java:316) [wicket-core-1.5.3.jar:1.5.3]
	at sun.reflect.GeneratedMethodAccessor144.invoke(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25) ~[na:1.6.0_29]
	at java.lang.reflect.Method.invoke(Method.java:597) ~[na:1.6.0_29]
	at org.apache.wicket.RequestListenerInterface.internalInvoke(RequestListenerInterface.java:260) [wicket-core-1.5.3.jar:1.5.3]
	at org.apache.wicket.RequestListenerInterface.invoke(RequestListenerInterface.java:241) [wicket-core-1.5.3.jar:1.5.3]
	at org.apache.wicket.request.handler.ListenerInterfaceRequestHandler.invokeListener(ListenerInterfaceRequestHandler.java:255) [wicket-core-1.5.3.jar:1.5.3]
	at org.apache.wicket.request.handler.ListenerInterfaceRequestHandler.respond(ListenerInterfaceRequestHandler.java:234) [wicket-core-1.5.3.jar:1.5.3]
	at org.apache.wicket.request.cycle.RequestCycle$HandlerExecutor.respond(RequestCycle.java:750) [wicket-core-1.5.3.jar:1.5.3]
	at org.apache.wicket.request.RequestHandlerStack.execute(RequestHandlerStack.java:64) [wicket-request-1.5.3.jar:1.5.3]
	at org.apache.wicket.request.cycle.RequestCycle.execute(RequestCycle.java:252) [wicket-core-1.5.3.jar:1.5.3]
	at org.apache.wicket.request.cycle.RequestCycle.processRequest(RequestCycle.java:209) [wicket-core-1.5.3.jar:1.5.3]
	at org.apache.wicket.request.cycle.RequestCycle.processRequestAndDetach(RequestCycle.java:280) [wicket-core-1.5.3.jar:1.5.3]
	at org.apache.wicket.protocol.http.WicketFilter.processRequest(WicketFilter.java:162) [wicket-core-1.5.3.jar:1.5.3]
	at org.apache.wicket.protocol.http.WicketFilter.doFilter(WicketFilter.java:218) [wicket-core-1.5.3.jar:1.5.3]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243) [catalina.jar:7.0.23]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210) [catalina.jar:7.0.23]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:224) [catalina.jar:7.0.23]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:169) [catalina.jar:7.0.23]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:472) [catalina.jar:7.0.23]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:168) [catalina.jar:7.0.23]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:98) [catalina.jar:7.0.23]
	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:928) [catalina.jar:7.0.23]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118) [catalina.jar:7.0.23]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:407) [catalina.jar:7.0.23]
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:987) [tomcat-coyote.jar:7.0.23]
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:539) ...

_Original issue: http://code.google.com/p/syncope/issues/detail?id=233_

**Comment by ilgrosso** [comment]
_From [chicchiricco](https://code.google.com/u/chicchiricco/) on December 05, 2011 08:17:07_

**Labels:** Milestone-Release-201112 Source-trunk Component-UI

**Comment by ilgrosso** [comment]
_From [chicchiricco](https://code.google.com/u/chicchiricco/) on December 05, 2011 08:17:30_

**Labels:** -Component-UI

**Comment by ilgrosso** [comment]
_From [chicchiricco](https://code.google.com/u/chicchiricco/) on December 06, 2011 03:40:34_

**Status:** Started  
  **Owner:** chicchiricco

**Comment by ilgrosso** [comment]
_From [chicchiricco](https://code.google.com/u/chicchiricco/) on December 06, 2011 05:31:00_

This issue was closed by revision r1288 .

**Status:** Fixed
